lib/wallaby/browser.ex

```diff
@@ -937,7 +937,10 @@ defmodule Wallaby.Browser do
   end
 
   defp request_url(path) do
-    base_url() <> path
+    base_url = base_url() |> String.trim_trailing("/")
+    path = path |> String.trim_leading("/")
```

Comment:
I think style wise piping when there's only one function call in the pipe is discouraged aka maybe rather `String.trim_leading(path, "/")` ?

Reply:
Good call. I'll update the code.